com.playeveryware.eos/Runtime/Core/EOSManager.cs

```diff
@@ -1176,11 +1176,16 @@ public void StartConnectLoginWithOptions(Epic.OnlineServices.Connect.LoginOption
                 connectInterface.Login(ref connectLoginOptions, null,
                     (ref Epic.OnlineServices.Connect.LoginCallbackInfo connectLoginData) =>
                     {
+                        if (connectLoginData.ResultCode != Result.Success)
+                        {
+                            print($"Connect login was not successful. ResultCode: {connectLoginData.ResultCode}", LogType.Error);
+                        }
```

Comment:
I recall an earlier conversation on this topic, but I do not recall the outcome: I believe that it's possible that a result code can be returned indicating that the user is already logged in (I believe the result code is something like `AlreadyConfigured`). _If that is the case_, then we need to change this conditional to be:

```c#
if (connectLoginData.ResultCode != Result.Success && connectLoginData.ResultCode != Result.AlreadyConfigured)
{
    /* ... */
}
```

Reply:
I'll create a situation that reproduces this message. The documentation mentions error codes, but not which are specifically used here.